.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
I put a bug in there myself (`./index.js` instead of `index.js`), please correct that as well. :P